packages/p/podman/package.yml

```diff
@@ -27,19 +30,32 @@ rundeps    :
     - netavark
     - passt
     - skopeo
-setup      : |
-    %patch -p1 -i $pkgfiles/catatonit.patch
 build      : |
     export CGO_CFLAGS="$CFLAGS"
     export GOFLAGS="-trimpath"
     export BUILDTAGS="seccomp varlink systemd"
 
     set -e
+
+    mkdir catatonit
+    tar --strip-components=1 -xf $sources/catatonit.tar.gz -C catatonit
+    pushd catatonit
+    autoreconf -fi
+    ./configure
```

Comment:
Out of curiosity, will our `%reconfigure` macro work here?

Reply:
done